run-tests-with-docker.sh

```diff
@@ -11,6 +11,7 @@ docker-compose run \
                --name verify-tests \
                -e TEST_ENV="${TEST_ENV:-"staging"}" \
                -e RUNNING_IN_DOCKER=true \
+               -e CUCUMBER_PUBLISH_QUIET=true \
```

Comment:
What does this do?

Reply:
It stops stdout like this popping out regularly:
````
┌──────────────────────────────────────────────────────────────────────────┐
│ Share your Cucumber Report with your team at https://reports.cucumber.io │
│                                                                          │
│ Command line option:    --publish                                        │
│ Environment variable:   CUCUMBER_PUBLISH_ENABLED=true                    │
│ cucumber.yml:           default: --publish                               │
│                                                                          │
│ More information at https://reports.cucumber.io/docs/cucumber-ruby       │
│                                                                          │
│ To disable this message, specify CUCUMBER_PUBLISH_QUIET=true or use the  │
│ --publish-quiet option. You can also add this to your cucumber.yml:      │
│ default: --publish-quiet                                                 │
└──────────────────────────────────────────────────────────────────────────┘
````